README.adoc

```diff
@@ -285,6 +285,14 @@ Put a file or directory into the remote host.
 |String, *Mandatory*
 |file or directory path on the remote node.
 
+|filterBy
+|String, *Optional*, Defaults to `name`.
```

Comment:
Does it support filtering by that other than name? I'm suggesting to condense these two to one property for now: filterByName, then document it's a regex based filter.

Reply:
I haven't tried all the options, but yes it does support all the params those are available on the File object, like `path`, `absolutePath`, `file`, `directory`,  `parent`, `parentFile` and so on.. 

Mostly all the params on the File object http://docs.groovy-lang.org/docs/latest/html/groovy-jdk/java/io/File.html